components/lib/calendar/Calendar.js

```diff
@@ -148,7 +148,6 @@ export const Calendar = React.memo(
                 const value = parseValueFromString(props.timeOnly ? rawValue.replace('_', '') : rawValue);
 
                 if (isValidSelection(value)) {
-                    isTypingRef.current = true;
```

Comment:
Because isTypingRef is not longer used you should remove the `isTypingRef` declaration at the top of the file